.jazzy.yml

```diff
@@ -0,0 +1,50 @@
+# jazzy --swift-version 2.2 -m Deferred -g "https://github.com/bignerdranch/Deferred" -a "Big Nerd Ranch" -u "http://bignerdranch.github.io" --module-version=2.0.0 -r "http://bignerdranch.github.io/Deferred/"
+author: "Big Nerd Ranch"
+author_url: "http://bignerdranch.github.io"
+copyright: "© 2014-2016 [Big Nerd Ranch](https://www.bignerdranch.com) under [MIT license](https://github.com/bignerdranch/Deferred/blob/master/LICENSE.txt)."
```

Comment:
Sorry, right. Clarification. The `copyright` stanza is included as Markdown in the footer of the docs.